source/includes/items/_create.md

```diff
@@ -0,0 +1,342 @@
+## Create An Item
+
+> `POST http://localhost:3000/api/v6/items`
+
+```ruby
+example_example_params = {
+  item: {
+    location_id: Integer,
+    condition_id: Integer,
+    status_id: Integer,
+    code: String,
+    usage: String ['in_use', 'unused'],
+    project: Text,
+    x: Decimal, 
+    y: Decimal,
+    purchase_date: Date, 
+    purchase_price_cents: Decimal,
+    room: String,
+    warranty: Integer, 
+    warranty_info: Text,
+    action_id: Integer,
+    installation_date: Date, 
+    life_expectancy: Float,
+    catalogue_item_id: ???,
```

Comment:
what do they need to submit here?

Reply:
Are you referring to the '???' next to catalogue item id? If so, that does bring light to a few extra endpoints we're going to need.
- Catalogues - index/show (by using client id)
- Categories - index/show (by using catalogue id)
- Types - index/show (by using category id)
- Catalogue Items - index/show (by using type id)

With those extra endpoints external API users will be able to find out what catalogue id they need when creating an item.

Reply:
@pacMakaveli mentioned this yesterday, as a catalogue API. so i guess the question is whether we point them directly to the catalogue API, or if we add end points in EDi that call the catalogue API?